spec/services/whats_app_adapter/three_sixty_dialog/file_fetcher_service_spec.rb

```diff
@@ -7,7 +7,7 @@
   describe '#call' do
     subject { -> { described_class.new(organization_id: organization.id, file_id: file_id).call } }
 
-    let(:organization) { create(:organization) }
+    let!(:organization) { create(:organization) }
```

Comment:
I've run:
```
git checkout origin/main -- spec/services/whats_app_adapter/three_sixty_dialog/file_fetcher_service_spec.rb
```

But this spec doesn't fail.

Instead, here and there, intermittent failures of other, unrelated specs.

@mattwr18 can you shine some light on this?

Reply:
So it didn't fail on my machine because I had a `ATTR_ENCRYPTED_KEY` set?